src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -365,7 +365,15 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
                             className={classnames('fa fa-fw')}
                             style={{ marginRight: 4 }}
                         />
-                        Compare
+                        <span
```

Comment:
@adamabeshouse i think better to solve this by using display:flex btn-group div.  This seems to prevent the wrapping in my experiment